azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/bindings/CosmosDBBinding.java

```diff
@@ -12,54 +12,81 @@
 import com.microsoft.azure.functions.annotation.CosmosDBOutput;
 import com.microsoft.azure.functions.annotation.CosmosDBTrigger;
 
-@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class CosmosDBBinding extends BaseBinding {
     public static final String COSMOS_DB_TRIGGER = "cosmosDBTrigger";
-    public static final String COSMOS_DB = "cosmosdb";
-
-    private String databaseName = "";
-
-    private String collectionName = "";
-
-    private String leaseCollectionName = "";
-
-    private String id = "";
-
-    private String sqlQuery = "";
-
-    private String connectionStringSetting = "";
-
-    private boolean createIfNotExists = false;
-
-    private boolean createLeaseCollectionIfNotExists = false;
+    public static final String COSMOS_DB = "cosmosDB";
+
+    private Integer collectionThroughput = null;
```

Comment:
Add comments to classify which are common attributes, which are trigger attributes, etc...